static/app/views/replays/detail/layout/focusTabs.tsx

```diff
@@ -40,6 +35,7 @@ function FocusTabs({className}: Props) {
       {Object.entries(getReplayTabs()).map(([tab, label]) =>
         label ? (
           <ListLink
+            data-test-id={label}
```

Comment:
The 'Error' badge got removed because of this it seems.

That's something we should try to overcome, because there'll be new or updated tabs again in the future and we'll want to put badges back.

Can we use the `tab` value instead?